server/src/main/java/org/elasticsearch/snapshots/SnapshotInfo.java

```diff
@@ -528,6 +637,16 @@ public XContentBuilder toXContent(final XContentBuilder builder, final Params pa
             builder.value(index);
```

Comment:
We don't really need this list anymore if we have the index details anyway?
I mainly care about this when thinking about the API response to get snapshots which now blows up in size of the response quite a bit.

Reply:
We don't necessarily have the index details, however, and I don't think we can remove stuff from the JSON response in a non-breaking change.

Reply:
Yea, I was just wondering if we really want to have redundant string list and then another map of the details that has all its keys in that list, that seemed weird and maybe worth a change in 8.x? I think we're good for now if we have a flag for these details as you suggest I think though.